extension/src/popup/ducks/settings.ts

```diff
@@ -599,3 +608,8 @@ export const settingsStateSelector = createSelector(
   settingsSelector,
   (settings) => settings.settingsState,
 );
+
+export const isNonSSLEnabledSelector = createSelector(
+  settingsSelector,
+  (settings) => !isMainnet(settings.networkDetails) || settings.isNonSSLEnabled,
```

Comment:
talking with the devx folks and some users on discord, it seems like the SSL check is annoying for a lot of devs testing local dapps on Testnet. So, we're only going to check the setting on non-Mainnet networks